docs/sdk/swap-widget/_category_.json

```diff
@@ -1,5 +1,5 @@
 {
   "label": "Swap Widget",
-  "position": 2,
```

Comment:
Was something else made position 2?

Reply:
This guide but it was arbitrary! I'll move it back!